core-java-modules/core-java-arrays-sorting/src/test/java/com/baeldung/sorted/operations/InsertElementInSortedArrayTest.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.sorted.operations;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class InsertElementInSortedArrayTest {
```

Comment:
Test names must match the convention in 
https://github.com/eugenp/tutorials/blob/ff0f7909e82fb25eba816aaf2db587c0100a4053/custom-pmd/src/main/java/com/baeldung/pmd/UnitTestNamingConventionRule.java#L18

```suggestion
public class InsertElementInSortedArrayUnitTest {
```